test/sql/test_external_file/R/test_parquet_optional_column_levels

```diff
@@ -0,0 +1,33 @@
+-- name: testParquetOptionalColumnLevels
```

Comment:
Can using c++ code instead, refer to `FileReaderTest::TestLateMaterializationAboutRequiredComplexType`

Reply:
Easily for other people who needs to refactor code.